espresso.py

```diff
@@ -156,8 +155,7 @@ def flash() -> None:
                 '0x20000', args.firmware,
             )
             if not success:
-                print_fail('Flashing failed. Maybe you need different parameters? Or you forgot "sudo"?\n')
-                sys.exit(1)
+                raise RuntimeError('Flashing failed. Maybe you need different parameters? Or you forgot "sudo"?')
```

Comment:
Maybe a little shorter and more 'commanding'?
```suggestion
                raise RuntimeError('Flashing failed. Use "sudo" and check your parameters.')
```
or 'Make sure to use "sudo"...'